src/template.txt

```diff
@@ -54,8 +54,8 @@ define(<%= imports %>, function(<%= args %>){
 
 			this.renderAsync().then(function(result){
 				var frag = result.fragment;
-				var head = document.head;
-				var body = document.body;
+				var head = document.getElementsByTagName("head")[0];
+				var body = document.getElementsByTagName("body")[0];
```

Comment:
`document.body` exists in IE8 so we only need this for the head.  Also I would do `document.head || document.getElementsByTagName("head")[0]` just to prevent the traversing of the entire dom when not needed.